modules/homer/maketagdirectory/meta.yml

```diff
@@ -3,23 +3,34 @@ description: Create a tag directory with the HOMER suite
 keywords:
   - peaks
   - bed
+  - bam
+  - sam
 tools:
   - homer:
       description: |
         HOMER (Hypergeometric Optimization of Motif EnRichment) is a suite of tools for Motif Discovery and next-gen sequencing analysis.
       documentation: http://homer.ucsd.edu/homer/
       doi: 10.1016/j.molcel.2010.05.004.
       licence: ["GPL-3.0-or-later"]
+  - samtools:
+      description: |
+        SAMtools is a set of utilities for interacting with and post-processing
+        short DNA sequence read alignments in the SAM, BAM and CRAM formats, written by Heng Li.
+        These files are generated as output by short read aligners like BWA.
+      homepage: http://www.htslib.org/
+      documentation: hhttp://www.htslib.org/doc/samtools.html
+      doi: 10.1093/bioinformatics/btp352
+      licence: ["MIT"]
 input:
   - meta:
       type: map
       description: |
         Groovy Map containing sample information
         e.g. [ id:'test', single_end:false ]
-  - bed:
+  - bam:
```

Comment:
runs on both `bed` and `bam`, right. What about renaming to `input`?

Reply:
I just saw you did that in #1445. I like input, but I also like when we use `reads` for raw reads in a fastq. I guess I'm also afraid of everything being named `input` :see_no_evil:

Reply:
fair enough :D, I think I started this originally because all the variantcallers said `meta,bam` but we want to use `cram`. In some places i think `input_aligned` is used, which I also like, gives a bit more description. Is there an overlap between `bam`/`bed` content wise?

Reply:
`aligned_reads`? `regions` in bed/bam case? I like `input_aligned` too.

I think sticking with bam here for now is fine until we get everyone's input on some naming conventions. I had to make this PR because using a `bed` while possible runs into memory issues and then homer silently crashes on some samples. So while it's possible, I don't want to encourage people to use it.

Reply:
ok sounds reasonable 👍

Reply:
https://github.com/nf-core/modules/issues/1450

Summarized here so we can follow up!